src/container/srv_target.c

```diff
@@ -2233,7 +2233,8 @@ ds_cont_tgt_snapshot_notify_aggregator(crt_rpc_t *source, crt_rpc_t *result,
 
 	out_source = crt_reply_get(source);
 	out_result = crt_reply_get(result);
-	out_result->tso_rc += out_source->tso_rc;
+	if (out_result->tso_rc >= 0 && out_source->tso_rc < 0)
```

Comment:
No change requested - just curious about the following. I imagine the test "out_result->tso_rc >= 0 ..." in this code is safest (compared to "out_result->tso_rc == 0").

Is it expected that tso_rc will always be 0 or less than 0 through the entire chain (dss_pool_thread_collective on each engine, the aggregated per-thread rc values from all threads in one engine, and the aggregated tso_rc value here across multple engines)? It seems that the code for thread collective and bcast corpc here takes the first observed error and assigns it to the aggregated rc value. So I guess in theory the test here could possibly be "(out_result->tso_rc == 0") && ..." ?